src/components/Subscribe/style.js

```diff
@@ -59,18 +62,22 @@ export const subscribe = theme => css`
 export const emailInput = theme => css`
   border: none;
   font-size: 3.2rem;
+  background-color: transparent;
+  border-radius: 0;
```

Comment:
put these to global style in `src/components/layout.js`

Reply:
done.